README.rst

```diff
@@ -1,14 +1,12 @@
-
-README for Pylint - https://pylint.pycqa.org/
-=============================================
+Pylint
+======
```

Comment:
If we include the readme in the doc this part is a little strange 😄

Reply:
Yeah that's true. Can we make the header a link? So that `Pylint` takes you to the documentation? I like how the link is so prominent. The `docs[passing]` button is not really the same.

Reply:
That's clever